_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,353 @@
+    protocol developers may want to keep in mind.
+
+- **Proposed service for storing, relaying, and broadcasting presigned transactions:**
+  Jacob Swambo sent a [request for comments][swambo rfc] to the Bitcoin-Dev
+  mailing list about creating software and a protocol for allowing
+  applications to store presigned transactions with third-party services
+  for later broadcast.  The software could also perhaps determine when to
+  broadcast the transactions based on certain conditions being met.  This
+  could be useful for software such as [vaults][topic vaults] and
+  [watchtowers][topic watchtowers].  Swambo asks anyone interested in
+  the idea, especially those interested in using it with their protocol, to
+  contact him.
+
+## Recently transcribed talks and conversations
+
+*[Bitcoin Transcripts][] is the home for transcripts of technical
+Bitcoin presentations and discussions. In this monthly feature, we
+highlight a selection of the transcripts from the previous month.*
+
+- **Magical Bitcoin:** Alekos Filini presented at LA BitDevs on [Magical
+  Bitcoin][], an in-development set of tools and libraries for onchain
+  wallet developers. He explained that coin selection logic and
+  transaction signing logic currently have to be rewritten multiple times
+  across multiple projects---a problem Magical Bitcoin aims to address
+  with modular, extensible, and peer-reviewed components.  A longer
+  term ambition is to provide a platform for building native
+  wallets and integrating them with existing projects.  Filini demoed
+  the current functionality, which includes a playground with a Policy
+  to [Miniscript][topic miniscript] compiler and some rudimentary
+  visualizations. It is written in Rust and leverages the
+  rust-miniscript library and the open source Esplora block explorer.
+  ([transcript][mb ts], [video][mb vid])
+
+- **Watchtowers and BOLT13:** Sergi Delgado appeared on [Potzblitz][] to
+  discuss the latest state of [watchtower][topic watchtowers]
+  development and a proposed watchtower protocol specification. He
+  explored the various tradeoffs when designing a watchtower and the
+  interplay between privacy requirements, accessibility, storage, and
+  fees charged.  Delgado is working on the watchtower implementation
+  [The Eye of Satoshi][teos] at Talaia Labs, which is aiming to be
+  compliant with the proposed BOLT13 (see [Newsletter #75][news75
+  watchtower bolt]).  Delgado also highlighted how watchtowers are
+  becoming increasingly critical in multiple settings such as Bitcoin
+  [vault][topic vaults] designs, statechains, and atomic swaps in addition to LN.
+  ([transcript][watchtowers ts], [video][watchtowers vid],
+  [slides][watchtowers slides])
+
+- **CoinSwap:** Aviv Milner presented on CoinSwap at the Wasabi Research
+  Club. He explained the property of covertness and how Chris Belcher’s
+  CoinSwap proposal (see [Newsletter #100][news100 coinswap]) provides
```

Comment:
coinswap? here and below, per style guide (unless this is a specific variant = proper noun)